tests/bin/typescript-demo-lib.test.ts

```diff
@@ -1,9 +1,64 @@
 import main from '@/bin/typescript-demo-lib';
+import { mockProcessStdout } from 'jest-mock-process';
```

Comment:
Always put external imports above internal imports. So move this ahead of line 1.